pkg/crd/markers/validation.go

```diff
@@ -175,6 +182,26 @@ type Default struct {
 	Value interface{}
 }
 
+// +controllertools:marker:generateHelp:category="CRD pruning"
```

Comment:
nit: can you stick this in the validation category?  it just makes it less weird to show in the docs (or we should come up with a general category for non-validation schema things, along with `listType` and such).  cc @apelisse

Reply:
I've used "CRD topology", wdyt?

Reply:
SGTM

Reply:
(or "CRD processing" or "CRD structure" -- whatever you think gets the point across)

Reply:
I thought about structure too. processing is super "imperative", it's such a bad word ;-)

Reply:
`CRD processing` fits best. It's not about structure, but about the way we process requests.

Reply:
updated